Core/src/main/java/org/tribuo/impl/IndexedArrayExample.java

```diff
@@ -364,6 +364,11 @@ public IndexedArrayExample<T> copy() {
         return new IndexedArrayExample<>(this);
     }
 
+    /**
+     * Unlike {@link ArrayExample#densify(List)} this method will throw {@link IllegalArgumentException}
+     * if one of the feature names is not present in this example's {@link ImmutableFeatureMap}.
+     * @param featureList A *sorted* list of feature names.
```

Comment:
the requirement that the input list be sorted should be more prominent in the docs

Reply:
Fixed.